Packs/Whois/Integrations/Whois/Whois_test.py

```diff
@@ -201,3 +201,20 @@ def test_ip_command(mocker):
              'Indicator': '4.4.4.4',
              'Score': 0,
              'Type': 'ip'}}
+
+
+def test_get_whois_ip_proxy_param(mocker):
+    """
+    Given:
+        - proxy address
+
+    When:
+        - running the get_whois_ip function
+
+    Then:
+        - Verify the function doesn't fail due to type errors
+    """
+    from Whois import get_whois_ip
+    mocker.patch.object(demisto, 'params', return_value={"proxy": "https://test.com"})
```

Comment:
shouldnt you mock `proxy_url` here?

Reply:
Not really, 
we test for type mismatch when trying to connect to the proxy server, it doesn't matter if the proxy server is valid.

Reply:
didnt mean the url itself, i mean the param you're currently mocking is the boolean one and not the string one which is used for the `ip` command